addStyles.js

```diff
@@ -128,18 +130,30 @@ function removeStyleElement(styleElement) {
 
 function createStyleElement(options) {
 	var styleElement = document.createElement("style");
-	styleElement.type = "text/css";
+	options.tagAttrs.type = "text/css";
+
+	attachTagAttrs(styleElement, options.tagAttrs);
 	insertStyleElement(options, styleElement);
 	return styleElement;
 }
 
 function createLinkElement(options) {
 	var linkElement = document.createElement("link");
-	linkElement.rel = "stylesheet";
+	options.tagAttrs.rel = "stylesheet";
+
+	attachTagAttrs(styleElement, options.tagAttrs);
 	insertStyleElement(options, linkElement);
 	return linkElement;
 }
 
+function attachTagAttrs(element, attrs) {
+	for(var key in attrs) {
```

Comment:
options.attrs is not an array but an key-value pair (object).
So forEach won`t work.